src/main/kotlin/app/revanced/cli/command/utility/InstallCommand.kt

```diff
@@ -44,20 +45,35 @@ internal object InstallCommand : Runnable {
                 }.install(Installer.Apk(apk, packageName))
             } catch (e: Exception) {
                 logger.severe(e.toString())
+                throw e
             }
 
             when (result) {
-                RootInstallerResult.FAILURE ->
+                RootInstallerResult.SUCCESS ->
+                    logger.info("Mounted the APK file")
+                RootInstallerResult.FAILURE -> {
                     logger.severe("Failed to mount the APK file")
-                is AdbInstallerResult.Failure ->
-                    logger.severe(result.exception.toString())
-                else ->
+                    throw Exception()
```

Comment:
raising an empty exception doesn't look right to me.

Reply:
Same as explained above - it's for control flow only.

Reply:
In this case, you can probably just return?

Reply:
Like above, we should continue this discussion under the [return code chain](https://github.com/ReVanced/revanced-cli/pull/365#discussion_r2051780152)